.gitignore

```diff
@@ -2,6 +2,8 @@
 ### GitHub repository .gitignore section
 ###
 
+composer.lock
```

Comment:
Hmm, I guess I could go either way on this one.  Not committing a lockfile at all means that each build can yield a different set of dependencies, depending on when it happened.  At the same time, we're already sometimes running composer update during the build (inside of the install-or-update script), so if that's possible anyway, then you're right - there's no reason to have a lockfile.  

I have to say though - this is definitely against best practices.  Lock file merge conflicts are definitely a huge pain, but they force us to resolve the exact dependencies we need locally so that every build of a single commit produces the same result.  This is super important in Multidev where you can get one thing on the build for your feature branch, and a completely other thing when the PR gets merged to master.  So anyway, that's my rant, but I'm open to whatever works for the team here.